Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-gem "rails", "7.1.3.4"
+gem "rails", "7.1.3.4" # Ideally version should be synced with Bouncer (activerecord)
```

Comment:
This is because they share a database

Reply:
Worth adding this to the comment/Readme?

Reply:
Yep - forgot to do that!